ui/components/MesherySettings.js

```diff
@@ -206,6 +206,10 @@ class MesherySettings extends React.Component {
     this.systemResetRef = React.createRef();
   }
 
+  componentDidMount() {
+    this.props.updatepagetitle({ title : "Settings" })
+  }
+
```

Comment:
Hey @Meyazhagan ,
Great work on the PR.
I had a some points to consider:
Is there any specific reason as to why this change was made inside the `MesherySettings.js` and not on `settings.js` inside the pages directory.
That seems like a better approach as there's already a line to updatepagepath there and would make it similar to:

https://github.com/meshery/meshery/blob/005fdd9e53fc4653f04b5a480b3f28c226069c11/ui/pages/system/connections.js#LL13C1-L15C9